README.md

```diff
@@ -135,14 +135,13 @@ Example command:
 
 <!-- FUTURE FEATURES -->
 
-<!-- ## 🔭 Future Features <a name="future-features"></a>
+## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[]**
-- [ ] **[]**
-- [ ] **[]**
-- [ ] **[]**
+- [ ] **[View From MVC]**
+- [ ] **[User Interface]**
+- [ ] **[Forms ]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to remove the guide note below the MIT license, on _line 172_.